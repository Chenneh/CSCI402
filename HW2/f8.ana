
==== Packets ====
p1: inter-arrival time should be 311.029ms
    time in Q1 should be 1432.044ms
    time in Q2 should be 0.019ms
    should request 2000ms of service time
    time in S1 should be 2018.754ms
    time in system should be 3450.858ms

p2: inter-arrival time should be 303.610ms
    time in Q1 should be 2952.869ms
    time in Q2 should be 0.012ms
    should request 2000ms of service time
    time in S2 should be 2006.260ms
    time in system should be 4959.163ms

p3: inter-arrival time should be 308.560ms
    time in Q1 should be 4392.227ms
    time in Q2 should be 0.013ms
    should request 2000ms of service time
    time in S1 should be 2007.468ms
    time in system should be 6399.724ms

p4: inter-arrival time should be 319.122ms
    time in Q1 should be 5816.715ms
    time in Q2 should be 0.013ms
    should request 2000ms of service time
    time in S2 should be 2018.695ms
    time in system should be 7835.438ms

p5: inter-arrival time should be 317.726ms
    time in Q1 should be 7245.246ms
    time in Q2 should be 0.033ms
    should request 2000ms of service time
    time in S1 should be 2019.325ms
    time in system should be 9264.623ms

p6: inter-arrival time should be 318.512ms
    time in Q1 should be 8718.255ms
    time in Q2 should be 0.013ms
    should request 2000ms of service time
    time in S2 should be 2017.744ms
    time in system should be 10736.028ms

p7: inter-arrival time should be 313.322ms
    time in Q1 should be 10124.396ms
    time in Q2 should be 0.015ms
    should request 2000ms of service time
    time in S1 should be 2017.558ms
    time in system should be 12141.983ms

p8: inter-arrival time should be 317.078ms
    time in Q1 should be 11558.731ms
    time in Q2 should be 0.017ms
    should request 2000ms of service time
    time in S2 should be 2018.461ms
    time in system should be 13577.220ms

==== Statistics ====
average packet inter-arrival time = 0.313619875
average packet service time = 2.015533125
average number of packets in Q1 = 3.24750744918108
average number of packets in Q2 = 8.39221769139167e-06
average number of packets in S1 = 0.501239499470731
average number of packets in S2 = 0.501118589371399
average time a packet spent in system = 8.545629625
standard deviation for time spent in system = 3.30707914315892
token drop probability = 0
packet drop probability = 0
