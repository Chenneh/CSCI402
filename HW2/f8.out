Emulation Parameters:
	number to arrive = 8
	lambda = 3.333
	mu = 0.5
	r = 3
	B = 7
	P = 5
00000000.000ms: emulation begins
00000311.029ms: p1 arrives, needs 5 tokens, inter-arrival time = 311.029ms
00000311.037ms: p1 enters Q1
00000353.681ms: token t1 arrives, token bucket now has 1 token 
00000614.639ms: p2 arrives, needs 5 tokens, inter-arrival time = 303.610ms
00000614.647ms: p2 enters Q1
00000691.011ms: token t2 arrives, token bucket now has 2 tokens
00000923.199ms: p3 arrives, needs 5 tokens, inter-arrival time = 308.560ms
00000923.208ms: p3 enters Q1
00001042.111ms: token t3 arrives, token bucket now has 3 tokens
00001242.321ms: p4 arrives, needs 5 tokens, inter-arrival time = 319.122ms
00001242.329ms: p4 enters Q1
00001394.394ms: token t4 arrives, token bucket now has 4 tokens
00001560.047ms: p5 arrives, needs 5 tokens, inter-arrival time = 317.726ms
00001560.056ms: p5 enters Q1
00001743.069ms: token t5 arrives, token bucket now has 5 tokens
00001743.081ms: p1 leaves Q1, time in Q1 = 1432.042ms, token bucket now has 0 token 
00001743.107ms: p1 enters Q2
00001743.126ms: p1 leaves Q2, time in Q2 = 0.019ms
00001743.133ms: p1 begin service at S1, requesting 2000ms of service
00001878.559ms: p6 arrives, needs 5 tokens, inter-arrival time = 318.512ms
00001878.568ms: p6 enters Q1
00002166.950ms: token t6 arrives, token bucket now has 1 token 
00002191.881ms: p7 arrives, needs 5 tokens, inter-arrival time = 313.322ms
00002191.889ms: p7 enters Q1
00002508.943ms: token t7 arrives, token bucket now has 2 tokens
00002508.959ms: p8 arrives, needs 5 tokens, inter-arrival time = 317.078ms
00002508.962ms: p8 enters Q1
00002845.104ms: token t8 arrives, token bucket now has 3 tokens
00003196.581ms: token t9 arrives, token bucket now has 4 tokens
00003567.507ms: token t10 arrives, token bucket now has 5 tokens
00003567.516ms: p2 leaves Q1, time in Q1 = 2952.867ms, token bucket now has 0 token 
00003567.519ms: p2 enters Q2
00003567.531ms: p2 leaves Q2, time in Q2 = 0.012ms
00003567.542ms: p2 begin service at S2, requesting 2000ms of service
00003761.887ms: p1 departs from S1, servie time = 2018.754ms, time in system = 3450.858ms
00003931.883ms: token t11 arrives, token bucket now has 1 token 
00004275.460ms: token t12 arrives, token bucket now has 2 tokens
00004621.164ms: token t13 arrives, token bucket now has 3 tokens
00004964.555ms: token t14 arrives, token bucket now has 4 tokens
00005315.426ms: token t15 arrives, token bucket now has 5 tokens
00005315.435ms: p3 leaves Q1, time in Q1 = 4392.225ms, token bucket now has 0 token 
00005315.438ms: p3 enters Q2
00005315.451ms: p3 leaves Q2, time in Q2 = 0.013ms
00005315.455ms: p3 begin service at S1, requesting 2000ms of service
00005573.802ms: p2 departs from S2, servie time = 2006.260ms, time in system = 4959.163ms
00005661.980ms: token t16 arrives, token bucket now has 1 token 
00006014.394ms: token t17 arrives, token bucket now has 2 tokens
00006367.876ms: token t18 arrives, token bucket now has 3 tokens
00006706.389ms: token t19 arrives, token bucket now has 4 tokens
00007059.035ms: token t20 arrives, token bucket now has 5 tokens
00007059.044ms: p4 leaves Q1, time in Q1 = 5816.713ms, token bucket now has 0 token 
00007059.048ms: p4 enters Q2
00007059.061ms: p4 leaves Q2, time in Q2 = 0.013ms
00007059.064ms: p4 begin service at S2, requesting 2000ms of service
00007322.923ms: p3 departs from S1, servie time = 2007.468ms, time in system = 6399.724ms
00007410.207ms: token t21 arrives, token bucket now has 1 token 
00007761.135ms: token t22 arrives, token bucket now has 2 tokens
00008110.916ms: token t23 arrives, token bucket now has 3 tokens
00008452.421ms: token t24 arrives, token bucket now has 4 tokens
00008805.290ms: token t25 arrives, token bucket now has 5 tokens
00008805.302ms: p5 leaves Q1, time in Q1 = 7245.244ms, token bucket now has 0 token 
00008805.307ms: p5 enters Q2
00008805.340ms: p5 leaves Q2, time in Q2 = 0.033ms
00008805.345ms: p5 begin service at S1, requesting 2000ms of service
00009077.759ms: p4 departs from S2, servie time = 2018.695ms, time in system = 7835.438ms
00009158.616ms: token t26 arrives, token bucket now has 1 token 
00009516.740ms: token t27 arrives, token bucket now has 2 tokens
00009869.373ms: token t28 arrives, token bucket now has 3 tokens
00010207.005ms: token t29 arrives, token bucket now has 4 tokens
00010596.813ms: token t30 arrives, token bucket now has 5 tokens
00010596.823ms: p6 leaves Q1, time in Q1 = 8718.253ms, token bucket now has 0 token 
00010596.827ms: p6 enters Q2
00010596.840ms: p6 leaves Q2, time in Q2 = 0.013ms
00010596.843ms: p6 begin service at S2, requesting 2000ms of service
00010824.670ms: p5 departs from S1, servie time = 2019.325ms, time in system = 9264.623ms
00010939.989ms: token t31 arrives, token bucket now has 1 token 
00011281.939ms: token t32 arrives, token bucket now has 2 tokens
00011629.218ms: token t33 arrives, token bucket now has 3 tokens
00011964.482ms: token t34 arrives, token bucket now has 4 tokens
00012316.276ms: token t35 arrives, token bucket now has 5 tokens
00012316.285ms: p7 leaves Q1, time in Q1 = 10124.394ms, token bucket now has 0 token 
00012316.288ms: p7 enters Q2
00012316.303ms: p7 leaves Q2, time in Q2 = 0.015ms
00012316.306ms: p7 begin service at S1, requesting 2000ms of service
00012614.587ms: p6 departs from S2, servie time = 2017.744ms, time in system = 10736.028ms
00012661.048ms: token t36 arrives, token bucket now has 1 token 
00013014.722ms: token t37 arrives, token bucket now has 2 tokens
00013350.511ms: token t38 arrives, token bucket now has 3 tokens
00013713.027ms: token t39 arrives, token bucket now has 4 tokens
00014067.682ms: token t40 arrives, token bucket now has 5 tokens
00014067.693ms: p8 leaves Q1, time in Q1 = 11558.729ms, token bucket now has 0 token 
00014067.697ms: p8 enters Q2
00014067.714ms: p8 leaves Q2, time in Q2 = 0.017ms
00014067.718ms: p8 begin service at S2, requesting 2000ms of service
00014333.864ms: p7 departs from S1, servie time = 2017.558ms, time in system = 12141.983ms
00016086.179ms: p8 departs from S2, servie time = 2018.461ms, time in system = 13577.220ms
00016086.332ms: emulation ends
	average packet inter-arrival time time = 0.31362
	average packet service time = 2.01553

	average number of packets in Q1 = 3.24751
	average number of packets in Q2 = 8.39221e-06
	average number of packets in S1 = 0.501239
	average number of packets in S2 = 0.501118
	average time a packet spent in system = 8.54563
	standard deviation for time spent in system = 3.30708

	token drop probability = 0
	packet drop probability = 0

