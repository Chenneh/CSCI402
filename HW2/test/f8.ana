
==== Packets ====
p1: inter-arrival time should be 307.093ms
    time in Q1 should be 1444.858ms
    time in Q2 should be 0.050ms
    should request 2000ms of service time
    time in S1 should be 2005.916ms
    time in system should be 3450.897ms

p2: inter-arrival time should be 317.653ms
    time in Q1 should be 2844.316ms
    time in Q2 should be 0.026ms
    should request 2000ms of service time
    time in S2 should be 2008.284ms
    time in system should be 4852.650ms

p3: inter-arrival time should be 318.719ms
    time in Q1 should be 4260.225ms
    time in Q2 should be 0.014ms
    should request 2000ms of service time
    time in S1 should be 2004.258ms
    time in system should be 6264.513ms

p4: inter-arrival time should be 317.909ms
    time in Q1 should be 5668.379ms
    time in Q2 should be 0.014ms
    should request 2000ms of service time
    time in S2 should be 2012.235ms
    time in system should be 7680.665ms

p5: inter-arrival time should be 315.958ms
    time in Q1 should be 7099.037ms
    time in Q2 should be 0.013ms
    should request 2000ms of service time
    time in S1 should be 2029.633ms
    time in system should be 9128.749ms

p6: inter-arrival time should be 316.671ms
    time in Q1 should be 8545.748ms
    time in Q2 should be 0.014ms
    should request 2000ms of service time
    time in S2 should be 2016.290ms
    time in system should be 10562.084ms

p7: inter-arrival time should be 321.611ms
    time in Q1 should be 9937.368ms
    time in Q2 should be 0.013ms
    should request 2000ms of service time
    time in S1 should be 2016.210ms
    time in system should be 11953.608ms

p8: inter-arrival time should be 315.947ms
    time in Q1 should be 11358.308ms
    time in Q2 should be 0.013ms
    should request 2000ms of service time
    time in S2 should be 2011.530ms
    time in system should be 13369.866ms

==== Statistics ====
average packet inter-arrival time = 0.316445125
average packet service time = 2.0130445
average number of packets in Q1 = 3.21718993186127
average number of packets in Q2 = 9.87326438860062e-06
average number of packets in S1 = 0.506619017580008
average number of packets in S2 = 0.506136170930481
average time a packet spent in system = 8.407879
standard deviation for time spent in system = 3.25361229387915
token drop probability = 0
packet drop probability = 0
