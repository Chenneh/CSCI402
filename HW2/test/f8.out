Emulation Parameters:
	number to arrive = 8
	lambda = 3.333
	mu = 0.5
	r = 3
	B = 7
	P = 5
00000000.000ms: emulation begins
00000307.093ms: p1 arrives, needs 5 tokens, inter-arrival time = 307.093ms
00000307.101ms: p1 enters Q1
00000351.149ms: token t1 arrives, token bucket now has 1 token 
00000624.746ms: p2 arrives, needs 5 tokens, inter-arrival time = 317.653ms
00000624.755ms: p2 enters Q1
00000701.105ms: token t2 arrives, token bucket now has 2 tokens
00000943.465ms: p3 arrives, needs 5 tokens, inter-arrival time = 318.719ms
00000943.474ms: p3 enters Q1
00001051.954ms: token t3 arrives, token bucket now has 3 tokens
00001261.374ms: p4 arrives, needs 5 tokens, inter-arrival time = 317.909ms
00001261.384ms: p4 enters Q1
00001401.271ms: token t4 arrives, token bucket now has 4 tokens
00001577.332ms: p5 arrives, needs 5 tokens, inter-arrival time = 315.958ms
00001577.360ms: p5 enters Q1
00001751.946ms: token t5 arrives, token bucket now has 5 tokens
00001751.959ms: p1 leaves Q1, time in Q1 = 1444.858ms, token bucket now has 0 token 
00001752.014ms: p1 enters Q2
00001752.064ms: p1 leaves Q2, time in Q2 = 0.050ms
00001752.074ms: p1 begin service at S1, requesting 2000ms of service
00001894.003ms: p6 arrives, needs 5 tokens, inter-arrival time = 316.671ms
00001894.028ms: p6 enters Q1
00002092.818ms: token t6 arrives, token bucket now has 1 token 
00002215.614ms: p7 arrives, needs 5 tokens, inter-arrival time = 321.611ms
00002215.624ms: p7 enters Q1
00002432.320ms: token t7 arrives, token bucket now has 2 tokens
00002531.561ms: p8 arrives, needs 5 tokens, inter-arrival time = 315.947ms
00002531.570ms: p8 enters Q1
00002782.258ms: token t8 arrives, token bucket now has 3 tokens
00003119.030ms: token t9 arrives, token bucket now has 4 tokens
00003469.062ms: token t10 arrives, token bucket now has 5 tokens
00003469.071ms: p2 leaves Q1, time in Q1 = 2844.316ms, token bucket now has 0 token 
00003469.075ms: p2 enters Q2
00003469.101ms: p2 leaves Q2, time in Q2 = 0.026ms
00003469.112ms: p2 begin service at S2, requesting 2000ms of service
00003757.990ms: p1 departs from S1, servie time = 2005.916ms, time in system = 3450.897ms
00003824.525ms: token t11 arrives, token bucket now has 1 token 
00004161.146ms: token t12 arrives, token bucket now has 2 tokens
00004502.629ms: token t13 arrives, token bucket now has 3 tokens
00004852.879ms: token t14 arrives, token bucket now has 4 tokens
00005203.691ms: token t15 arrives, token bucket now has 5 tokens
00005203.699ms: p3 leaves Q1, time in Q1 = 4260.225ms, token bucket now has 0 token 
00005203.703ms: p3 enters Q2
00005203.717ms: p3 leaves Q2, time in Q2 = 0.014ms
00005203.720ms: p3 begin service at S1, requesting 2000ms of service
00005477.396ms: p2 departs from S2, servie time = 2008.284ms, time in system = 4852.650ms
00005554.627ms: token t16 arrives, token bucket now has 1 token 
00005893.816ms: token t17 arrives, token bucket now has 2 tokens
00006243.772ms: token t18 arrives, token bucket now has 3 tokens
00006595.718ms: token t19 arrives, token bucket now has 4 tokens
00006929.738ms: token t20 arrives, token bucket now has 5 tokens
00006929.763ms: p4 leaves Q1, time in Q1 = 5668.379ms, token bucket now has 0 token 
00006929.766ms: p4 enters Q2
00006929.780ms: p4 leaves Q2, time in Q2 = 0.014ms
00006929.804ms: p4 begin service at S2, requesting 2000ms of service
00007207.978ms: p3 departs from S1, servie time = 2004.258ms, time in system = 6264.513ms
00007279.174ms: token t21 arrives, token bucket now has 1 token 
00007628.283ms: token t22 arrives, token bucket now has 2 tokens
00007978.549ms: token t23 arrives, token bucket now has 3 tokens
00008327.137ms: token t24 arrives, token bucket now has 4 tokens
00008676.372ms: token t25 arrives, token bucket now has 5 tokens
00008676.397ms: p5 leaves Q1, time in Q1 = 7099.037ms, token bucket now has 0 token 
00008676.419ms: p5 enters Q2
00008676.432ms: p5 leaves Q2, time in Q2 = 0.013ms
00008676.448ms: p5 begin service at S1, requesting 2000ms of service
00008942.039ms: p4 departs from S2, servie time = 2012.235ms, time in system = 7680.665ms
00009024.133ms: token t26 arrives, token bucket now has 1 token 
00009373.768ms: token t27 arrives, token bucket now has 2 tokens
00009740.351ms: token t28 arrives, token bucket now has 3 tokens
00010089.904ms: token t29 arrives, token bucket now has 4 tokens
00010439.767ms: token t30 arrives, token bucket now has 5 tokens
00010439.776ms: p6 leaves Q1, time in Q1 = 8545.748ms, token bucket now has 0 token 
00010439.780ms: p6 enters Q2
00010439.794ms: p6 leaves Q2, time in Q2 = 0.014ms
00010439.797ms: p6 begin service at S2, requesting 2000ms of service
00010706.081ms: p5 departs from S1, servie time = 2029.633ms, time in system = 9128.749ms
00010789.329ms: token t31 arrives, token bucket now has 1 token 
00011125.394ms: token t32 arrives, token bucket now has 2 tokens
00011474.461ms: token t33 arrives, token bucket now has 3 tokens
00011813.179ms: token t34 arrives, token bucket now has 4 tokens
00012152.984ms: token t35 arrives, token bucket now has 5 tokens
00012152.992ms: p7 leaves Q1, time in Q1 = 9937.368ms, token bucket now has 0 token 
00012152.996ms: p7 enters Q2
00012153.009ms: p7 leaves Q2, time in Q2 = 0.013ms
00012153.012ms: p7 begin service at S1, requesting 2000ms of service
00012456.087ms: p6 departs from S2, servie time = 2016.290ms, time in system = 10562.084ms
00012497.305ms: token t36 arrives, token bucket now has 1 token 
00012833.155ms: token t37 arrives, token bucket now has 2 tokens
00013182.686ms: token t38 arrives, token bucket now has 3 tokens
00013531.822ms: token t39 arrives, token bucket now has 4 tokens
00013889.870ms: token t40 arrives, token bucket now has 5 tokens
00013889.878ms: p8 leaves Q1, time in Q1 = 11358.308ms, token bucket now has 0 token 
00013889.881ms: p8 enters Q2
00013889.894ms: p8 leaves Q2, time in Q2 = 0.013ms
00013889.897ms: p8 begin service at S2, requesting 2000ms of service
00014169.222ms: p7 departs from S1, servie time = 2016.210ms, time in system = 11953.608ms
00015901.427ms: p8 departs from S2, servie time = 2011.530ms, time in system = 13369.866ms
00015901.529ms: emulation ends

Statistics:

	average packet inter-arrival time time = 0.31644512
	average packet service time = 2.0130445

	average number of packets in Q1 = 3.2171899
	average number of packets in Q2 = 9.873261e-06
	average number of packets in S1 = 0.50661902
	average number of packets in S2 = 0.50613617
	average time a packet spent in system = 8.407879
	standard deviation for time spent in system = 3.2536123

	token drop probability = 0
	packet drop probability = 0

